### PR TITLE
Update deploy.sh to handle missing pagerduty secret and update temporary JH route to direct to traefik

### PR DESCRIPTION
**Pull request by maulikjs**
Jira Link: https://issues.redhat.com/browse/RHODS-1630

### CODE REVIEW

**Review by anishasthana — COMMENTED** [review]
Can you rebase on main?

**Comment by maulikjs** [comment]
@anishasthana done

File: monitoring/jupyterhub-route.yaml [hunk 1]
```diff
@@ -7,11 +7,11 @@ metadata:
   namespace: redhat-ods-applications
 spec:
   port:
-    targetPort: 8080-tcp
+    targetPort: api
```
Comment (anishasthana) [inline monitoring/jupyterhub-route.yaml:10]: What's going on here?

Comment (maulikjs) [reply monitoring/jupyterhub-route.yaml:10]: It doesn't really matter, it's just a placeholder route.

**Comment by anishasthana** [comment]
Maulik is working on deploying something for us to look at.

**Comment by crobby** [comment]
afaict, this is working

**Review by crobby — APPROVED** [review]

**Pull request merged**